### PR TITLE
Fix styling of Koa vs Express docs

### PR DESCRIPTION
**Pull request by shime**
* split sentences
* add commas where needed
* fix various grammar errors
* replace "node.js" with "Node"
* replace "koa" with "Koa"

### CODE REVIEW

**Comment by jonathanong** [comment]
sorry, we don't merge style-only PRs. it's not worth the git diff.

can you edit this PR to only fix: 

- grammatical errors (i can't tell what they are from this diff)
- koa => Koa

**Comment by niftylettuce** [comment]
Hey @shime nice to see you here 🙌  can you fix the PR per Jonathan's comments?

**Comment by shime** [comment]
@niftylettuce 👋  thanks for reminding me, just made the fixes per Jonathan's comments.

File: docs/koa-vs-express.md [hunk 1]
```diff
@@ -33,10 +33,10 @@ THIS DOCUMENT IS IN PROGRESS. THIS PARAGRAPH SHALL BE REMOVED WHEN THIS DOCUMENT
   and less error-prone to write, for the entire stack, not
   just the end application code.
 
-  Typically many middleware would
+  Typically many middlewares would
```
Comment (fl0w) [inline docs/koa-vs-express.md]: -1

Comment (jonathanong) [reply docs/koa-vs-express.md]: yes, only requested change

**Review by 3imed-jaberi — COMMENTED** [review]
@shime, can you resolve the conflict then change 'node' >>>> 'Node' !!

**Review by 3imed-jaberi — COMMENTED** [review]
@jonathanong, you can approve this PR and merge it 💯 !

**Review by 3imed-jaberi — COMMENTED** [review]
✅

**Review by 3imed-jaberi — APPROVED** [review]
🚀

**Review by fengmk2 — APPROVED** [review]

**Pull request merged**